Rounding decimal degrees to dms as expected

bedubs opened the pull request:
I also have a test file I can include if you want to use it, but delivering the expected value.

bedubs committed:
Rounding decimal degrees to dms as expected

I also have a test file I can include if you want to use it, but delivering the expected value.

mradamcox commented:
Looks good. I don't have time to test this fully right now, so can you modify it a little to make it return exactly the same object as before? It seems like right now it would work, but I'd rather be safe about it. Later it would be good to revisit and clean that map-tools file up because I had some unused stuff in it, and how it is called in the app [here for example](https://github.com/mradamcox/ead/blob/master/ead/media/js/resource-report.js#L93).

bedubs committed:
Changed back to returning generic object

bedubs committed:
Update ConvertDDToDMS seconds

Update ConvertDDToDMS with precision parameter that takes an argument to determine the number of decimal places to be returned on the seconds property. If precision is not specified in the function call, the value defaults to one.

mradamcox commented:
Great, this all look good!
